tests/scenarios/package.json

```diff
@@ -30,25 +30,25 @@
     "broccoli-persistent-filter": "^3.1.2",
     "ember-bootstrap": "^4.6.3",
     "ember-cli-3.16": "npm:ember-cli@~3.16.0",
```

Comment:
I'm curious, why keep 3.16 and drop 3.20?

Reply:
Purely to keep the test suite smaller. We still support both, we're just testing the older one of the two and assuming it's unlikely that the newer one is broken if LTS releases on either side of it are working.